frontend/tests/features/crudCollection.test.ts

```diff
@@ -0,0 +1,42 @@
+import { TEST_ENV, TEST_URL } from "tests/common/constants";
```

Comment:
maybe the file name can just be `Collection.test.ts`? Adding `crud` makes me wonder what non-CRUD tests we're expecting to write and if they should live in a separate file or just in the same file. Personally it feels all Collection related tests can live in this file, whether it's CRUD or not. What do you think?

Thank you!